_includes/header.html

```diff
@@ -28,19 +27,16 @@
 			<nav>
 				<div class="nav-bar">
 					<div class="module left">
-						<a href="https://fossasia.org/">
+						<a href="./#welcome">
```

Comment:
I think you haven't pulled latest changes. As there should be no diff here as this is already present in the existing codebase. Reference:https://github.com/fossasia/gci17.fossasia.org/blob/e0e4d070976777a939e7489b69188ec5f6276283/_includes/header.html#L30